src/run_releasenotes_check.py

```diff
@@ -0,0 +1,94 @@
+import logging
+import os
+import subprocess
+from typing import Any, List
+
+from pytablewriter import MarkdownTableWriter
+
+from checkout_workflow.checkout_args import CheckoutArgs
+from git.git_repository import GitRepository
+from manifests.input_manifest import InputComponentFromSource, InputManifest
+from system import console
+from system.temporary_directory import TemporaryDirectory
+
+# in format yyyy-mm-dd, example 2022-07-26
+gitLogDate = os.getenv('GIT_LOG_DATE')
+# Github issue to add the generated markdown table as a new comment
+gitIssueNumber = os.getenv('GIT_ISSUE_NUMBER')
+# Boolen if set to null, just prints on the console.
+addComment = os.getenv('ADD_COMMENT', default="true")
+# Token used to add comment on an issue
+userToken = os.getenv('GITHUB_TOKEN')
+
+
+def main(gitLogDate: str, addComment: str) -> int:
+    value_matrix = []  # type: List[Any]
+    args = CheckoutArgs()
+    console.configure(level=args.logging_level)
+    manifest = InputManifest.from_file(args.manifest)
+    writer = MarkdownTableWriter(
+        table_name=f"{manifest.build.name} Commits(after {gitLogDate}) & Release Notes info",
+        headers=["Repo", "Branch", "CommitID", "Release Notes"],
+        value_matrix=value_matrix
+    )
+    with TemporaryDirectory(keep=True, chdir=True) as work_dir:
```

Comment:
Using `keep=True` the dir will persist which I dont think is of any issue as such? Default is false so just remove it?

Reply:
Changed to `false` @gaiksaya

Reply:
If you want you can add a param to control this behavior like build.py or assemble.py.
I am ok to set this as false.

The defaults are in the src/system/temp*.